tasks/infra/delete-osd-task.yaml

```diff
@@ -12,8 +12,8 @@ spec:
         limits:
           cpu: '100m'
           memory: 64Mi
-      image: 'quay.io/trepel/kuadrant-pipelines:latest'
-      imagePullPolicy: Always
+      image: quay.io/kuadrant/testsuite-pipelines-tools:latest
+      imagePullPolicy: IfNotPresent
```

Comment:
ifNotPresent won't pull the image if latest is pointing on a different tag, as far as I know

Reply:
I'm not sure if that is what you want to achieve here. `IfNotPresent` will pull the tag only if it's not present locally, it is used for the static tags (`v1.0`, `v1.2`). `latest` most of the time is a floating tag that always changes what digest it is pointed at. Therefore `latest` tag won't update if you change its digest and one of the latest versions already is present on a cluster.

Even k8s defaults to `ifNotPresent` if image pull policy is not present, except if image has the :latest tag, in that case `Always` is set as a default value